Automattic/jetpack
# Dashboard: Prompt connection instead of upgrade when user not connected

Fixes 1164141197617539-as-1200266637330561

Fixes #19703 

#### Changes proposed in this Pull Request:

Sites with a site-only connection were seeing upgrade prompts on the AAG dashboard, but upgrade was not possible without a user connection. This PR changes that so that users see a connection prompt instead.

#### Jetpack product discussion


pbtFFM-TV-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Create a new Jetpack site.
2. In an incognito/private window, open the Jetpack site and log into wp-admin.
3. Go to connect Jetpack and choose “Continue with WordPress.com”
4. In the popup, create a new account (or log into an account you know has no sites attached to it).
5. When you are redirected to the authorization page, ignore it and don’t complete authorization. 
6. Visit the Jetpack dashboard in wp-admin, and verify that all prompts on that page are to connect and not to upgrade.
